src/common/components/Recaptcha/Recaptcha.tsx

```diff
@@ -31,63 +36,74 @@ export const Recaptcha = ({
   buttonDisabled,
   buttonLabel,
   $translate,
-  $log
+  $log,
+  recaptchaKey
 }: RecaptchaProps): JSX.Element => {
-  const { executeRecaptcha } = useGoogleReCaptcha();
+
+  const [ready, setReady] = useState(false)
+  const [grecaptcha, setGrecaptcha] = useState(window.grecaptcha)
+  let timesIntervalRun = 0
+
+  useEffect(() => {
```

Comment:
Should there be a failsafe, just in case an error happens with recaptcha, and `window.grecaptcha` doesn't get defined?

Like after so many seconds, we set it to ready? But I guess maybe not, as the bots could just wait.

Reply:
It could be worth it; so far in my testing I haven't run into an issue with this, but we also don't want to prevent donors from giving. If we do a failsafe, we'd just call `onSuccess` instead of setting `ready` to true.